hudi-platform-service/hudi-metaserver/hudi-metaserver-server/src/main/java/org/apache/hudi/metaserver/store/RelationalDBBasedStorage.java

```diff
@@ -105,8 +106,7 @@ public Long getTableId(String db, String tb) throws MetaserverStorageException {
 
   @Override
   public String createNewTimestamp(long tableId) throws MetaserverStorageException {
-    // todo: support SSS
-    SimpleDateFormat sdf = new SimpleDateFormat("yyyyMMddHHmmss");
+    SimpleDateFormat sdf = new SimpleDateFormat("yyyyMMddHHmmss:SSS");
```

Comment:
can we reuse `org.apache.hudi.common.table.timeline.HoodieInstantTimeGenerator#MILLIS_INSTANT_TIME_FORMATTER` here? so to make formats aligned

Reply:
miss this, will fix